pkg/acsengine/testdata/vnet/kubernetesvnet.json

```diff
@@ -49,6 +49,11 @@
       "clientPrivateKey": "clientPrivateKey",
       "kubeConfigCertificate": "kubeConfigCertificate",
       "kubeConfigPrivateKey": "kubeConfigPrivateKey"
+    },
+    "kubeNetworkConfig": {
+      "kubeDnsServiceIp": "172.17.1.10",
```

Comment:
Well I think old files won't have kubeNetworkConfig section. In this case I usethe old hardcoded values as defaults in pkg/acsengine/defaults.go
So users with old template files are still able to create the deployment templates.